stock_measuring_device/static/src/js/measuring_wizard.esm.js

```diff
@@ -0,0 +1,32 @@
+import {registry} from "@web/core/registry";
+
+export const measuringWizardNotificationService = {
+    dependencies: ["action", "bus_service", "orm"],
+    start(env, {action, bus_service, orm}) {
+        this.action = action;
+        this.bus_service = bus_service;
+        this.orm = orm;
+        const self = this;
+        this.bus_service.subscribe("notify_measuring_wizard_screen", (notification) => {
+            self.notifyMeasuringWizardScreen(notification).then((result) => {
+                return result;
+            });
```

Comment:
```suggestion
            self.notifyMeasuringWizardScreen(notification);
```

This doesn't seem to be doing anything.
What returns here is the arrow function set to process `result`, but it's not processing anything